src/main/java/org/spongepowered/api/item/inventory/ItemStack.java

```diff
@@ -85,7 +85,7 @@ static ItemStack of(ItemType itemType, int quantity) {
      * @return The empty ItemStack
      */
     static ItemStack empty() {
-        return builder().itemType(ItemTypes.NONE).build();
+        return builder().itemType(ItemTypes.AIR).build();
```

Comment:
Should put this in a separate pr to prevent merge conflicts

Reply:
He should rebase on top of the latest changes.